Examples/Io/Root/include/ActsExamples/Io/Root/RootBFieldWriter.hpp

```diff
@@ -113,10 +113,11 @@ class RootBFieldWriter {
 
     // Get the underlying mapper of the InterpolatedBFieldMap
     auto mapper = cfg.bField->getMapper();
+    auto grid = mapper.getGrid();
 
     // Access the minima and maxima of all axes
-    auto minima = mapper.getMin();
-    auto maxima = mapper.getMax();
+    auto minima = grid.minPosition();
+    auto maxima = grid.maxPosition();
```

Comment:
This had to be adjusted because it now reports the interpolation domain, rather than the raw grid extent.